ASIM/dev/ASimTester/filteringTest/ASimFilteringTest.py

```diff
@@ -307,81 +307,155 @@ def get_values_for_dynamic_tests(self, rows):
                     break
         return values
         
-    
+
     # Performing assertions for dynamic tests with parameter filtering. Values for the parameter are taken from values_list
-    def dynamic_tests_assertions(self, param_name, query_definition, column_name_in_table, values_list, no_filter_response):
-        pass #TODO will be added in next PR
+    def dynamic_tests_assertions(self, param_name, query_definition, column_name_in_table, values_list, no_filter_rows):
+        val_str = create_values_string(values_list)
+        query_with_filter = query_definition + create_execution_strings_with_one_parameter(param_name,f"dynamic([{val_str}])" ,column_name_in_table)
+        filtered_response = self.send_query(query_with_filter)
+        filtered_rows = filtered_response.tables[0].rows
```

Comment:
>filtered_rows [](http://example.com/codeflow?start=8&length=13)

nit:
remove this, and replace with:
response_rows_length = len(filtered_response.tables[0].rows)